docker/build-tools/Dockerfile

```diff
@@ -49,16 +49,16 @@ ENV GH_VERSION=2.68.1
 ENV GOCOVMERGE_VERSION=b5bfa59ec0adc420475f97f89b58045c721d761c
 ENV GOIMPORTS_VERSION=v0.24.0
 # When updating the golangci version, you may want to update the common-files config/.golangci* files as well.
-ENV GOLANGCI_LINT_VERSION=v1.64.6
+ENV GOLANGCI_LINT_VERSION=v2.0.0
```

Comment:
Happy to update this but we need non-trivial work in every repo to accommodate this change before we can merge this

Reply:
Ok. I'll back this one out before I merge and we can do it as a separate fix